pkg/driver/mysql/mysql.go

```diff
@@ -192,7 +193,17 @@ func (drv *Driver) DumpSchema(db *sql.DB) ([]byte, error) {
 	}
 
 	schema = append(schema, migrations...)
-	return dbutil.TrimLeadingSQLComments(schema)
+	schema, err = dbutil.TrimLeadingSQLComments(schema)
+	if err != nil {
+		return nil, err
+	}
+	return TrimAutoincrementValues(schema), nil
+}
+
+// TrimAutoincrementValues removes AUTO_INCREMENT values from MySQL schema dumps
+func TrimAutoincrementValues(data []byte) []byte {
```

Comment:
I think this can start with a lowercase letter since it does not need to be exported?